tests/common/adapters/__snapshots__/test_fhir_api.ambr

```diff
@@ -0,0 +1,11 @@
+# name: test_inmemory_fhir_api_validate
+  <class 'dict'> {
+    'issue': <class 'list'> [
+    ],
+    'resourceType': 'OperationOutcome',
+    'text': <class 'dict'> {
+      'div': '<p> it went fine bro </p>',
+      'status': 'generated',
+    },
+  }
+---
```

Comment:
I don't know what is this doing here? @simonvadee @tevariou any idea?

Reply:
must be removed (remains from the merge I guess)